src/ipc/bridge.cc

```diff
@@ -1309,9 +1309,9 @@ export default exports.default ?? undefined
     // URLs (socket:...)
     auto moduleTemplate =
 R"S(
+import module from '{{url}}'
 export * from '{{url}}'
-const exports = await import('{{url}}');
-export default exports.default ?? undefined
+export default module
```

Comment:
we can save the template to a variable to reuse in both line 1223 and 1310

Reply:
done